### PR TITLE
Introduced datatables filtering for HPO phenotypes

### LINKED ISSUE #1447: Phenotype search is down [dev version]
Entering search terms fires no autocomplete, and searching anyway causes crashes - perhaps on a string decompose expecting an auto-complete style string?

### PR DESCRIPTION
**Pull request by northwestwitch**
Use dataTables.js and not a form search to filter phenotypes. fix #1447

**How to test**:
- Check phenotypes page and make sure filter works

**Expected outcome**:
- Page should work

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready to be reviewed!

**Comment by moonso** [comment]
I had a look at the code but think @dnil is better suited to review this

File: scout/server/blueprints/phenotypes/templates/phenotypes/hpo_terms.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {% extends "layout_bs4.html" %}
 
+{% block css %}
+{{ super() }}
+  <link rel="stylesheet" href="https://cdn.datatables.net/1.10.20/css/jquery.dataTables.min.css">
```
Comment (dnil) [inline scout/server/blueprints/phenotypes/templates/phenotypes/hpo_terms.html]: Nice!

File: scout/server/blueprints/phenotypes/controllers.py [hunk 2]
```diff
@@ -1,20 +1,15 @@
 # -*- coding: utf-8 -*-
 
-def hpo_terms(store, query = None, limit = None):
+def hpo_terms(store):
```
Comment (dnil) [inline scout/server/blueprints/phenotypes/controllers.py]: That's one long page, but it seems to work just fine! I don't think ppl will be reloading this page too often either.

File: tests/server/blueprints/phenotypes/test_phenotypes_controllers.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from scout.server.blueprints.phenotypes import controllers
+
+def test_hpo_terms(real_adapter):
```
Comment (dnil) [inline tests/server/blueprints/phenotypes/test_phenotypes_controllers.py]: :+1:

**Pull request merged**